lxd/cluster/config/config.go

```diff
@@ -312,6 +313,7 @@ var ConfigSchema = config.Schema{
 	"images.default_architecture":    {Validator: validate.Optional(validate.IsArchitecture)},
 	"images.remote_cache_expiry":     {Type: config.Int64, Default: "10"},
 	"instances.nic.host_name":        {Validator: validate.Optional(validate.IsOneOf("random", "mac"))},
+	"instances.placement_scriptlet":  {Validator: validate.Optional(scriptletLoad.InstancePlacementValidate)},
```

Comment:
Should we make this `instances.placement.scriptlet` instead? This would in theory allow for `instances.placement.webhook` or `instances.placement.strategy` should we want to offer alternatives.